phpcs.xml.dist

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards">
+	<description>Apply WordPress Coding Standards to all Core files</description>
+
+	<!--
+	phpcbf -.-standard=phpcs8.xml.dist -.-report-summary -.-report-source -.-report-full=./20170926-11-phpcbf-all-incl-new-sniffs.txt -.-basepath=I:/000_GitHub/WP_test_suite/
+	-->
```

Comment:
The above 3 line comment should be removed, or updated to reference a _suggested_ command line example, e.g. `phpcbf --standard=phpcs.xml.dist --report-summary --report-source`